packages/web-api/src/WebClient.ts

```diff
@@ -182,6 +188,7 @@ export class WebClient extends Methods {
     rejectRateLimitedCalls = false,
     headers = {},
     teamId = undefined,
+    attachOriginalToWebAPIRequestError = true,
```

Comment:
We should also add this option to the `WebClientOptions` interface definition on line 61 of this file, ideally with a JSDoc (so that developers can get some basic information about this option as they develop in their IDE). You can reuse the JSDoc you added on line 170 for this if you wish, but I wouldn't link to the repo issue. We want to document this in a way that is clear to developers without having to open a link.